dvc/utils/serialize/_py.py

```diff
@@ -0,0 +1,89 @@
+import ast
```

Comment:
I believe this is not cross-python-version compatible, right? Not a blocker though.

Reply:
ast uses the compiler of the installed python. As I understand it, ast is compatible https://stackoverflow.com/a/26656125